base_frame: "base_link"
odom_frame: "odom"
encoder_topic_name: "/roboteq_driver/encoder_count"
command_srv: "/roboteq_driver/dualchannel_command_service"
gear_ratio: 24.69
radius: 0.105
wheelbase: 1.0
rate: 5
ppr: 1024
encoder_max: 65536
encoder_min: -65536
